.scalafmt.conf

```diff
@@ -0,0 +1,21 @@
+version = "3.5.3"
+runner.dialect = scala211
```

Comment:
2.11 or 2.12 ? Project is gonna support both, thus you chosen the smaller one out of them..?

Reply:
let's change it to `runner.dialect = scala212`